script/data-harmonizer/index.js

```diff
@@ -1214,6 +1226,19 @@ let DataHarmonizer = {
 	  if (field.string_serialization) {
 		guidance.push('Pattern hint: ' + field.string_serialization);
 	  }
+	  const hasMinValue = field.minimum_value != null;
+	  const hasMaxValue = field.maximum_value != null;
+	  if (hasMinValue || hasMaxValue) {
```

Comment:
I think min and max is inclusive? So text has to be a bit more verbose ... equal to or greater than ... etc.   "... inclusively between ..."

Reply:
Done.